wifiphisher.py

```diff
@@ -155,7 +155,7 @@ def do_GET(self):
                                )
                 log_file.close()
             self.path = "index.html"
-        self.path = "%s/%s" % (PHISING_PAGE, self.path)
+        self.path = "%s/%s" % ("access-point-pages/"+args.style, self.path)
```

Comment:
You need to check that the concatenated path exists, otherwise the script should return an error.

Also, please make sure to surround the plus with spaces.